packages/sakuli-cli/src/commands/run-command.function.spec.ts

```diff
@@ -8,13 +9,12 @@ import { mockPartial, mockRecursivePartial } from "sneer";
 import { runCommand } from "./run-command.function";
 import { Argv, CommandModule } from "yargs";
 import { testExecutionContextRenderer } from "../cli-utils/test-execution-context-renderer.function";
-import * as commons from "@sakuli/commons";
+import * as ensurePathModule from "@sakuli/commons/dist/fs/ensure-path.function";
```

Comment:
Was it intentional to import from the `dist` folder?

Reply:
Yep, this is required unfortunately because tsc js does not allow property redefinition anymore. This is way more clean and ensures module immutability but also breaks the test setup for some modules. This is a quick fix. One could discuss to export ensure path directly. This would clean up the dist reference but for a test case, this is not as important, IMHO.

Reply:
Thanks for the clarification :+1: I agree with you that it is not as important to clean up for one test.